static/js/Header.jsx

```diff
@@ -50,7 +50,7 @@ class Header extends Component {
           { Sefaria._siteSettings.TORAH_SPECIFIC ?
           <a className="home" href="/" >{logo}</a> : null }
           <a href="/texts" className="textLink"><InterfaceText context="Header">Texts</InterfaceText></a>
-          <a href="/topics" className="textLink"><InterfaceText>Topics</InterfaceText></a>
+          <a href="/topics" className="textLink"><InterfaceText text={{en:"Explore", he: "נושאים"}}/></a>
```

Comment:
Better to use context="Header". This will avoid creating a new object on each render. See line 52 for an example.